clang/test/CodeGen/integer-wrap.c

```diff
@@ -0,0 +1,77 @@
+// Check that -fsanitize=signed-integer-wrap instruments with -fwrapv
```

Comment:
Have you seen `llvm/utils/update_cc_test_checks.py` before?  Honestly, ALL new tests to clang (or llvm) involving codegen should strive to use that (or `update_any_test_checks.py` which I believe will now determine the correct underlying `update_*_test_checks.py` to use).

Reply:
I must be doing something wrong because it suggest I remove all my `Check: ...`'s and replcae them with `Check: {{.*}}`

Reply:
I believe @tschuett was saying that to be confident we are instrumenting the arithmetic (and the correct operands therein) we should check the overflow bit usage in the branch instruction. The only way I could think of doing this was capturing the call to the overflow intrinsic which returns the value and the overflow bit. Is there a way to minimize my CHECKs?

Reply:
> I must be doing something wrong because it suggests I remove all my Check: ...'s and replace them with Check: {{.*}}

Huh, indeed. I can repro. Not sure how to fix, but in digging around I found clang/test/codegen/integer-overflow.c.  Your newly added test file clang/test/CodeGen/integer-wrap.c is highly redundant of that. Why don't you remove clang/test/CodeGen/integer-wrap.c and simply add another RUN line to clang/test/codegen/integer-overflow.c?

Reply:
Gotcha, I did just that in [5497e8b](https://github.com/llvm/llvm-project/pull/80089/commits/5497e8bc6849bf64c1158ff16b4aa04fd9141920). Thanks Nick!